package.json

```diff
@@ -46,18 +46,19 @@
     "lint": "samples lint",
     "generate": "node ./scripts/generate",
     "pretest": "npm run lint && node ./scripts/clean coverage",
-    "unit-test": "ava --verbose -T 30s functions/**/test/**/*.test.js",
+    "unit-test": "ava --verbose -T 30s **/test/*.test.js",
     "unit-cover": "nyc --cache npm test && nyc report --reporter=html",
-    "system-test": "ava --verbose -T 30s vision/system-test/**/*.test.js",
+    "system-test": "ava --verbose -T 30s **/system-test/*.test.js",
     "system-cover": "nyc --cache npm run system-test && nyc report --reporter=html",
     "test": "npm run unit-test && npm run system-test",
-    "cover": "nyc --cache npm run test && nyc report --reporter=html"
+    "cover": "nyc --cache npm run test && nyc report --reporter=html",
+    "update-dependencies": "find . -name \"package.json\" -not -path \"*/node_modules/*\" -execdir sh -c \"ncu -u && ! git diff --quiet --exit-code . && ls . | grep -i test > /dev/null && yarn upgrade && ava -T 2m --fail-fast ./*est/*.test.js && git add package.json yarn.lock\" \\;"
```

Comment:
I think we can replace `ls . | grep -i test > /dev/null && yarn upgrade && ava -T 2m --fail-fast ./*est/*.test.js` with `yarn upgrade && yarn test -- --fail-fast`